app/assets/stylesheets/base.scss

```diff
@@ -0,0 +1,61 @@
+a {
+  &.disabled {
+    color: #ccc;
+    cursor: not-allowed;
+    text-decoration: none;
+  }
+}
+
+body {
+  font-family: 'Source Sans Pro', 'Arial Unicode MS', Helvetica, sans-serif;
+}
+
+a, a:hover, a:focus {
+  text-decoration: none;
+  border-bottom: 1px dotted $sul-link-color-border;
+}
+
+h1 {
+  font-family: 'dejavu_sansextralight';
+  color: $sul-h1-font-color;
+  padding: 10px 0;
+  letter-spacing: -.05em;
+}
+
+h2 {
+  letter-spacing: 0;
+  line-height: 1.2em;
+  margin: .5em 0;
+  color: $cardinal-red;
+  border-bottom: 1px solid $sul-h2-border-color;
+}
+
+h3 {
+  font-weight: 400;
+  line-height: 1.3em;
+  margin: 0 0 .4em 0;
```

Comment:
`.4` should be written with a leading zero as `0.4`<br>Shorthand form for property `margin` should be written more concisely as `0 0 .4em` instead of `0 0 .4em 0`